### ISSUE TITLE
Invalid argument

### ISSUE THREAD

**Issue by CtrlAltDefeat94** [pkkid/python-plexapi#276]
When trying: `plex.library.section("Music").albums()`

```
File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\library.py", line 623, in albums
    return self.fetchItems(key)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\base.py", line 147, in fetchItems
    return self.findItems(data, cls, ekey, **kwargs)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\base.py", line 163, in findItems
    item = self._buildItemOrNone(elem, cls, initpath)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\base.py", line 86, in _buildItemOrNone
    return self._buildItem(elem, cls, initpath)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\base.py", line 78, in _buildItem
    return ecls(self._server, elem, initpath)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\base.py", line 47, in __init__
    self._loadData(data)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\audio.py", line 174, in _loadData
    Audio._loadData(self, data)
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\audio.py", line 33, in _loadData
    self.lastViewedAt = utils.toDatetime(data.attrib.get('lastViewedAt'))
  File "C:\Program Files (x86)\Python36\lib\site-packages\plexapi\utils.py", line 176, in toDatetime
    value = datetime.fromtimestamp(int(value))
OSError: [Errno 22] Invalid argument
```

**Comment by Hellowlol** [comment]
You need to check the xml. It seems it to return junk.

**Comment by CtrlAltDefeat94** [comment]
How exactly do I do this?

**Comment by CtrlAltDefeat94** [comment]
I have been able to narrow down the problem to a few albums.
I am unsure why it crashes there, but if it could be of any help, I could provide the files that cause the crashes.

**Comment by Hellowlol** [comment]
the value of lastViewedAt is outside the range of localtime so it fails. We dont need the files. Just check the xml response. I cant remember where you can find it, i think there was a dropdown button where could click on something to get the xml up. Ill check it laster or you can google it.

**Comment by CtrlAltDefeat94** [comment]
Ok, I found where you can look it up (for future reference, it's under Get Info -> View XML)

Of the album that causes this issue, there's at least one song where lastViewedAt does not exist. 
After playing all the songs in that album, all songs will have the lastViewedAt value (verified it) and the crash doesn't occur anymore.

**Comment by Hellowlol** [comment]
Good :)

**Comment by CtrlAltDefeat94** [comment]
Wouldn't this require some sort of fail safe in the code or something?
I mean, I was able to figure out why it crashed and fixed it for this particular album, but apparently I've got plenty in my library that are missing this value.

**Comment by Hellowlol** [comment]
We could ofc add some random value, but that would just hide the fact that you need to rescan/readd the library.
I guess i could add some kind of warning of some sort so we cover everything.

**Comment by CtrlAltDefeat94** [comment]
If it's not a bug in the library, as I'm sure I wouldn't be the first to run into it if it were, what _should_  fix this issue?
My library is as up to date as can be(at least I think so), I just ran another scan files and update metadata, both are completed and I still get the issue on other albums/songs than the one I fixed earlier.

**Comment by CtrlAltDefeat94** [comment]
I think there's a need for something to catch this. 
Apparently lastViewedAt can be None. The code crashing because something is None (even if it's because you need to rescan/readd the library(which I did, but didn't help)), doesn't sound like good practice.

**Comment by CtrlAltDefeat94** [comment]
Fixed the issue locally myself so it no longer crashes.

**Comment by Hellowlol** [comment]
Ill add a proper fix later.

**Comment by Hellowlol** [comment]
@pkkid How should we handles this? Should we just set the date to None or add a datetime from epoch?

**Comment by CtrlAltDefeat94** [comment]
Locally, I set it to the start of epoch.
I'm not sure how dependent other parts of the library are on this value, but that looked like the safest approach.

**Comment by D34thPi3c3** [comment]
Is there any update planned because of this error? I have the same problem. At the moment i use "try and except" in my own program, however i would like to print out every episode, also the series that havent a "lastViewAt".

**Comment by jrcalvert** [comment]
I am seeing this also - the value passed to utils.toDateTime is negative, which errors under windows
I haven't delved into what the value should be, but it works if set value to 0 if < 0

**Comment by Tomme** [comment]
Thrown up a quick fix (#345). Feel free shoot down but I see limited issues simply ignoring these "junk" negative epoch values.
